ops/marketplace-tf/vars.tfvars

```diff
@@ -17,9 +17,9 @@ bacalhau_accept_networked_jobs = true
 
 bacalhau_otel_collector_endpoint = "http://analytics.bacalhau.tech:4318"
 
-bacalhau_requester_api_token = "token_for_requester_api"
-bacalhau_compute_api_token = "token_for_compute_api"
+bacalhau_requester_api_token = "password"
```

Comment:
did you mean to check in these as defaults? Terraform offers a feature to generate a random string (or this should be left blank and required)

Reply:
If these values are blank a random token will automatically be generated and used. I commit this since typing `password` was easier than the former.

Reply:
I hate that we auto-generate tokens and ask the users to go through the logs or to a local file on desk to get whatever token was generated. We should either allow unsecure access by default, or fail if no token was provided. but we shouldn't auto generate tokens

Reply:
i think i'm ok as well if the token goes into a defined file somewhere. Not logs. This feels like best practice to me.